spec/book_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
```

Comment:
- [ OPTIONAL BUT HIGHLY RECOMMENDED ]Hello! 🛠️ When you include require 'rspec' at the top of your Ruby test file, it's a good practice to add the RSpec gem to your project's Gemfile. This approach offers a more convenient way for anyone using your code to set up and run tests. 📦